### ISSUE TITLE
LibriSpeech Recipe not converging; loss explodes

### ISSUE THREAD

**Issue by FredericGodin** [facebookresearch/wav2letter#168]
I've been struggling for the last few weeks with getting the Recipe for LibriSpeech to work.
The loss is roughly 30 for a number of steps but then explodes.
TER always stays around 99%.

Was someone able to reproduce the results?
What could be the issue?

**Comment by vineelpratap** [comment]
The params mentioned for Librispeech recipe was using 8 gpus to train. 

If you are using different number of GPUs, you would have to play around with "-lr" and "-lrcrit" params to make it converge since the "effective" batchsize per iteration is different. 

Assuming you make it work, we also typically do learning rate annealing to get the model converge even better. 

Hope it helps !
